dotcom-rendering/src/web/browser/bootCmp/init.ts

```diff
@@ -63,27 +63,78 @@ const init = async (): Promise<void> => {
 
 	onConsentChange((consentState: ConsentState) => {
 		if (!consentState) return;
-
 		// Register changes in consent state with Ophan
+
+		/*
+			(Date: February 2022. Author: Pascal. This is a work in progress...)
+
+			### General Context
+
+			We are sending consent data from the client side to the datalake through Ophan, using Ophan Component Events.
+
+			We are using Ophan Component Events because consent events do not have a first class type in the Ophan pipeline
+			(this may change in the future)
+
+			At the moment we are using MANAGE_CONSENT as the default action while we develop this code.
+
+			### Encoding Conventions
+
+			At the moment, the payload is essentially contained in the `labels` attribute (an array of strings) of the
+			component object. We are going to encode consent data for more than one jusridiction. To make this unambiguous
+			we are going to use a specific convention.
+
+			Each element of the array represents a particular attribute. Note that the collection of attributes is different
+			from one jurisdiction to another. To have an unambiguous reading we prefix each attribute value with a code
+			that helps identify the attribute type.
+
+			An element of the array is then always of the form "<code>:<value>"
+
+			Codes:
+				01: Specify the jurisdiction. Accepted values: "TCF.v2" and "CCPA"
```

Comment:
Can we use string identifiers that at least hint at the contained value? I think i'll be looking up what they mean whenever I come to look at this data. Definitely keep the documentation explaining the detail though.

A consistent string length for the ID is a good idea though as it helps with removing the prefix identifier when writing SQL.

Reply:
Sure. Sure. Let me do that :)

Reply:
Discussed with @shtukas, this is fine for now.

Reply:
So... as discussed, let's try and run with the codes for the moment while we are in prototyping stage.